portal_pages/templates/portal_pages/case_study_index_page.html

```diff
@@ -49,6 +49,8 @@ <h1 class="cushion">{{ self.title }}</h1>
                         {% endif %}
 
                     </div>
+                    <br />
+                    <a class="button" href="{% routablepageurl self "submit" %}">Submit Your Own Case Study</a>
```

Comment:
I think this should say Submit Your own Story..."Case Study" I think is an internal name not exposed yet as what these are called on the public side? Public side refers to these as Stories I think pretty consistently...